snooty/test_product_landing.py

```diff
@@ -0,0 +1,171 @@
+from pathlib import Path
```

Comment:
I think these tests present a risk in a sense of over-testing work that isn't actually addressed by the PLP additions -- IMO you should only be testing the new directives that are not tested elsewhere. Otherwise more work is just introduced if there's ever a deliberate change to some of these other directives.